js/ui/html_editor/utils/image_uploader_helper.js

```diff
@@ -407,21 +463,25 @@ class FileStrategy extends BaseStrategy {
         return this.config.fileUploadMode === 'both';
     }
 
+    validate(e) {
```

Comment:
Let's rename ```e``` to ```event```

The ```e``` is a known name by all, but the ```event``` leaves no chance of misunderstanding.
Also, name variable with one letter name isn't a good practise (magic number / magic letter)